Xamarin.Forms.Platform.Android/FastRenderers/ImageElementManager.cs

```diff
@@ -50,30 +50,34 @@ async static void OnElementChanged(object sender, VisualElementChangedEventArgs
 		async static void OnElementPropertyChanged(object sender, PropertyChangedEventArgs e)
 		{
 			var renderer = (sender as IVisualElementRenderer);
-			var ImageElementManager = (IImageController)renderer.Element;
-			if (e.PropertyName == ImageElementManager.SourceProperty?.PropertyName)
+			var ImageElementManager = (IImageElement)renderer.Element;
+			var imageController = (IImageController)renderer.Element;
+
+			if (e.PropertyName == Image.SourceProperty.PropertyName ||
```

Comment:
The differing name of the Bindable property between Image and Button was the original reason that I exposed the BP's through the interface.  This class is generalized and applied to Image,Button, and ImageButton.  I just used the PropertyName on Button for most of these since that'll just be the same across all three controls.  Though that does feel a little brittle it's probably fine